### DISCUSSION TITLE
Some Microsoft Store packages don't show up in winget

### DISCUSSION THREAD

**Discussion by tygoee** [microsoft/winget-cli#4730] [General] [answered]
Some Microsoft Store packages don't show up in winget, for example `9NBLGGH2JHXJ` [(link)](https://www.microsoft.com/store/productId/9NBLGGH2JHXJ?ocid=pdpshare), which is the Minecraft Bedrock app. These are packages that are still in people's libraries, but don't show on the top of search results (they are kind of indirectly installed via the launcher since recently). Those apps are however still needed to have a desktop shortcut (which is why I install them)

**Comment by mdanish-kh** [comment, marked as answer]
As @denelon mentioned in https://github.com/microsoft/winget-cli/discussions/2894#discussioncomment-4823877:

> The Microsoft Store has its own policies for providing packages to the WinGet CLI. In general, apps need to be rated "e" for everyone and "free".

I believe the package mentioned isn't "free", correct?

**Reply by tygoee** [reply]
Yes, that is probably the reason. Is it also not possible to sign into winget with a microsoft account to solve it?

**Reply by mdanish-kh** [reply]
That request is tracked in:

- https://github.com/microsoft/winget-cli/issues/908
- https://github.com/microsoft/winget-cli/issues/1833

Adding your 👍 to the issues helps raising their priority
